### PR TITLE
Added CLI command for printing notifications

### PR DESCRIPTION
**Pull request by merlimat**

### CODE REVIEW

File: cmd/client/notifications/cmd.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+	RunE:  exec,
+}
+
+func exec(cmd *cobra.Command, args []string) error {
+	client, err := common.Config.NewClient()
+	if err != nil {
+		return err
+	}
+
+	defer client.Close()
```
Comment (nahguam) [inline cmd/client/notifications/cmd.go:23]: This swallows errors (and below)? Do we care?

File: cmd/client/notifications/cmd.go [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+	notifications, err := client.GetNotifications()
+	if err != nil {
+		return err
+	}
+
+	defer notifications.Close()
+
+	for notification := range notifications.Ch() {
```
Comment (nahguam) [inline cmd/client/notifications/cmd.go:32]: Does this clean close on interrupt?

Comment (merlimat) [reply cmd/client/notifications/cmd.go:32]: yes, it exits on Ctrl-C

File: cmd/client/notifications/cmd.go (hunk 1 above)
Comment (merlimat) [reply cmd/client/notifications/cmd.go:23]: I think it should be ok. It just runs and receives notifications until it's stopped

**Pull request merged**